settings.py

```diff
@@ -0,0 +1,41 @@
+import logging
+#########################################
+#										#
+#										#
+#			Global Variables			#
+#										#
+#										#
+#########################################
+
+
+PREFIX = "::"
+TOKEN = "your_token"
+AUTHOR_ID=289426079544901633
```

Comment:
Shouldn't be `AUTHORS_ID` ? :roll_eyes:

Reply:
It's rather meant to be the ID of the one running the script. I'll change the name so that it's more appropriate.

Reply:
Okay :D